### ISSUE TITLE
BUG : Extra "b." Prefix Added to Custom

### ISSUE THREAD

**Issue by fabioginzel** [rudderlabs/rudder-sdk-flutter#131]
**Description**:
After upgrading to `rudder_sdk_flutter: ^2.5.1`, I've noticed that when using the `traits.put` method to set custom traits, an additional "b." prefix is prepended to the trait keys.

**Steps to Reproduce**:
1. Install `rudder_sdk_flutter: ^2.5.1`.
2. Implement the following code snippet:
```dart
traits.put('cpf', user.cpf ?? '');

**Comment by desusai7** [comment]
Hey @fabioginzel, 

Thank you for reporting this issue, can you please try adding the below proguard rules to your proguard file and let us know if this solves the issue ?

```
-keep class * implements com.google.gson.JsonSerializer { *; }
-keepclassmembers class com.rudderstack.android.sdk.core.RudderContext { java.util.Map customContextMap; }
-keepclassmembers class com.rudderstack.android.sdk.core.RudderTraits { java.util.Map extras; }
```

**Comment by desusai7** [comment]
We are exposing these rules as part of the Android SDK as well to ensure this kind of issue does not happen in the future. We already raised a PR for it [here](https://github.com/rudderlabs/rudder-sdk-android/pull/321)

**Comment by desusai7** [comment]
@fabioginzel, did you get a chance to try the new rules ?

**Comment by desusai7** [comment]
@fabioginzel, we've released version 1.20.0 of Rudderstack's Android SDK with the Proguard rules bundled in it. Can you please try upgrading / re-installing the Flutter SDK so that it fetches the latest version of Android SDK and then check if this issue is solved for you?

**Comment by desusai7** [comment]
Hey @fabioginzel, 

We've released Rudderstack Flutter SDK version 2.6.0 and this version makes sure that the Android SDK version of at least 1.20.0 is used and Android SDK version > 1.20.0 comes with built-in pro guard rules to fix this kind of issue. 

Please upgrade to version 2.6.0 of the Flutter SDK and let us know if you are having any issues.

**Comment by desusai7** [comment]
Hey @fabioginzel, 

Closing this issue for now and feel free to open this again, if this issue isn't resolved yet.
